CLI references non-existing module

Hi !

Tried using tartifacts instead of tar.gz for more flexibility but alas, the CLI does not seem to run properly when I try. I think it's tied to https://github.com/blond/tartifacts/blob/master/bin/tartifacts#L10 :

 `const loadPatterns = require('../lib/load-patterns');`

The module `load-patterns` does not seem to exist anymore, possibly replaced by what is now defined in the file `patterns.js`. If such is indeed the case, could you update the CLI ? And if not, well I'll gladly accept any help for the issue I have when I try to create my tarball : 

> \> tartifacts --include="dist/**" --dest="dist/tarball.tar.gz"
> 
> module.js:341
>     throw err;
>     ^
> 
> Error: Cannot find module '../lib/load-patterns'

Thanks !

-pl

--- a/bin/tartifacts
+++ b/bin/tartifacts
@@ -7,7 +7,7 @@ const path = require('path');
 const meow = require('meow');
 const tartifacts = require('../');
 
-const loadPatterns = require('../lib/load-patterns');
+const loadPatterns = require('../lib/patterns').loadPatterns;
 
 const cli = meow(`
     Usage

